### ISSUE TITLE
shift-queue manual targeting

### ISSUE THREAD

**Issue by capt-phurious** [beyond-all-reason/Beyond-All-Reason#859]
It would be nice if long range units like berthas would cycle between the various shift-queue targeting locations. Right now it seems it will move to a location only if there is a valid target and it has been destroyed. I feel that it should move to the next targeting location regardless of whether any preexisting conditions are required. Perhaps an option to tweak this could be created in the settings menu for those who feel otherwise about the issue.

**Comment by PtaQQ** [comment]
Yes please, afaik there already exists a solution for this in Spring, just has to be enabled for LRPC. Its how nukes work afaik.

**Comment by SethDGamre** [comment]
what exactly would be the behavior desired here? Fire a shot, move to next queued target, fire a shot, etc?

**Comment by SethDGamre** [comment]
It's likely the thing Ptaq mentioned is in a widget or less likely a unitdef entry. Could also possibly be a gadget. A good start would be to search "corsilo" and see where it pops up in the repo that seems related to attack behavior.
I think it's a good first issue because it's probably adding the name of the LRPC in the same place this behavior is defined for the nuke silos
